packages/fxa-auth-server/config/index.ts

```diff
@@ -1135,6 +1135,11 @@ const conf = convict({
       env: 'CLIENT_ADDRESS_DEPTH',
       default: 3,
     },
+    remoteAddressChainOverride: {
```

Comment:
Just out of interest, why does this get duplicated?

Reply:
That was part of the oauthServer configs - there's a clientAddressDepth param there as well (though unused as of now). I've removed this since the updated approach re-using the existing onPreAuth does not need this second instance.